docs/website/docs/examples/qdrant_zendesk/index.md

```diff
@@ -0,0 +1,172 @@
+---
+title: Similarity Searching with dlt and Qdrant
+description: Learn how to use the dlt source, Zendesk and dlt destination, Qdrant to conduct a similarity search on your tickets data.
+keywords: [similarity search, example]
+---
+import Header from '../_examples-qdrant-header.md';
+
+<Header
+    intro="In this tutorial, you will learn how to do use dlt to store your
+    vectorized Zendesk tickets data in the dlt destination; Qdrant. You can
+    use Qdrant's vectorization and similarity searching capabilities on your tickets data,
+    while using dlt as a medium to automate your pipeline."
+    slug="qdrant_zendesk"
+    run_file="qdrant-snippets"
+/>
+
+Qdrant has been introduced as a new [destination](https://dlthub.com/docs/dlt-ecosystem/destinations/qdrant) for dlt, adding onto our list of high-performance vector databases. Similar to our guide on using the [Zendesk source to extract ticket data and store it into weaviate](https://dlthub.com/docs/walkthroughs/zendesk-weaviate#configuration), this article drafts a similar system to map the vectorized ticket data from Zendesk to Qdrant!
```

Comment:
this example has how-to style https://dlthub.com/docs/walkthroughs/zendesk-weaviate
but we need example style https://dlthub.com/docs/examples/incremental_loading/

1. add bullet points what we learn with this example
2. delete almost all text
3. keep the output with query results

Reply:
Great work @hibajamal. I agree with @AstrakhantsevaAA this section needs to be shortened and rephrased. I like the link to the Weaviate how-to.